src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -76,7 +76,22 @@ Sentry does not know if a local variable that looks like a credit card number ac
 
 ## Selectors
 
-Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data.
+Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data. A few examples:
+
+* `**` to scrub everywhere
+* `$error.value` to scrub in the exception message
+* `$message` to scrub the event-level log message
+* `extra.'My Value'` to scrub the key `My Value` in "Additional Data"
+* `extra.**` to scrub everything in "Additional Data"
```

Comment:
Why is this `extra` and not `$extra` like `$error` and `$message`?

Reply:
Because `$error` is a shortcut for `exception.*.values` and `$message` is a shortcut for `logentry.formatted` while `extra` is already short enough. There's a section further down which explains what `$error` expands to, but this section is just here for copy-pasting, not for deep understanding.

Reply:
Talked a bit off-GH about this.

Perhaps we can be more explicit about long forms and shortcuts ($ prefix).

Maybe we can turn this into a table?!


long form | short form | explanation of what it does
----------|------------|--------------------------
`exception.values.*.value` | `$error.value` | scrub error/exception message
... | ... | ...